PWGHF/vertexingHF/AliAnalysisTaskSEDvsRT.cxx

```diff
@@ -874,8 +877,18 @@ Double_t AliAnalysisTaskSEDvsRT::CalculateRTVal(AliAODEvent* esdEvent)
          TList *listMin = (TList*)regionsMinMaxReco->At(1);
 
          trackRTval = (listMax->GetEntries() + listMin->GetEntries()) / fAveMultiInTrans; //sum of transverse regions / average
-	 fNChargedInTrans->Fill(listMax->GetEntries() + listMin->GetEntries());
+	       fNChargedInTrans->Fill(listMax->GetEntries() + listMin->GetEntries());
          fHistPtLead->Fill(LeadingPt);
+         Double_t Randomizer = gRandom->Uniform();
```

Comment:
Maybe it would be good to add a data member and a setter for the seed to have a reproducible result

Reply:
This is just for early stage testing, so reproducibility is not required. But I will consider your suggestion for future iterations, thanks! :)

Reply:
Hello @fgrosa . So can you please approve this version for now?

Reply:
I approve it for now, but reproducibility is an important point, so please don't forget to implement it!